### PR TITLE
read parentNode on attached in order to support dynamically created elements

### LINKED ISSUE #22: paper-dialog-scrollable cannot be created using Javascript
```
var l = document.createElement("link");
l.setAttribute("href","https://rawgit.com/Download/polymer-cdn/master/lib/paper-dialog/paper-dialog.html");
l.setAttribute("rel","import");
document.head.appendChild(l);

var l = document.createElement("link");
l.setAttribute("href","https://rawgit.com/Download/polymer-cdn/master/lib/paper-dialog-scrollable/paper-dialog-scrollable.html");
l.setAttribute("rel","import");
document.head.appendChild(l);
```

```
var pd = document.createElement("paper-dialog");
var pds = document.createElement("paper-dialog-scrollable");

pd.appendChild(document.createElement("h2"))
pd.appendChild(pds)


document.body.innerHTML = "";
document.body.appendChild(pd);
```

Causes `Uncaught TypeError: Cannot set property 'sizingTarget' of null` in paper-dialog-scrollable attached because the dialogElement value has apparently been fetched and cached in the ready phase, which happens before attach and when parentNode is null.

### PR DESCRIPTION
**Pull request by valdrinkoshi**
Fixes #22 by reading `parentNode` on `attached`

### CODE REVIEW

**Comment by cdata** [comment]
This looks great. It seems like it would be really easy to add a test for regressions. Would you consider adding one?

File: paper-dialog-scrollable.html [hunk 1]
```diff
@@ -126,6 +123,9 @@
 
     attached: function() {
       this.classList.add('no-padding');
+      // read parentNode on attached because if dynamically created,
+      // parentNode will be null on creation.
+      this.dialogElement = this.dialogElement || this.parentNode;
```
Comment (cdata) [inline paper-dialog-scrollable.html]: `this.parentNode` -> `Polymer.dom(this).parentNode`

**Comment by valdrinkoshi** [comment]
@cdata done

**Comment by cdata** [comment]
LGTM :+1:

**Pull request merged**